benchmarks/maxtext_v5p_model_configs.py

```diff
@@ -0,0 +1,107 @@
+"""
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+"""Shared Benchmark config for v6e orchestrations."""
+
+import os.path
+from benchmarks import xla_flags_library
+from benchmarks.maxtext_trillium_model_configs import MaxTextModel, _add_to_model_dictionary
+
+
+v5p_model_dict = {}
+
+deepseek_v3_fsdp = _add_to_model_dictionary(
+    v5p_model_dict,
+    MaxTextModel(
+        model_name="deepseek_v3_fsdp",
+        model_type="deepseek3-671b",
+        tuning_params={
+            "per_device_batch_size": 4,
+            "max_target_length": 8192,
+            "ici_fsdp_parallelism": -1,
+            "ici_expert_parallelism": 1,
+            "remat_policy": "custom",
+            "decoder_layer_input": "offload",
+            "gcs_metrics": True,
+            "use_iota_embed": True,
+            "dataset_path": "gs://max-datasets-rogue",
+            "dataset_type": "synthetic",
+            "reuse_example_batch": 1,
+            "enable_checkpointing": False,
+            "skip_first_n_steps_for_profiler": 5,
+            "profiler_steps": 5,
+            "profiler": "xplane",
+            "sa_block_q": 2048,
+            "sa_block_q_dkv": 2048,
+            "sa_block_q_dq": 2048,
+            "megablox": False,
+            "sparse_matmul": False,
+            "capacity_factor": 1.0,
+            "tokenizer_type": "huggingface",
+            "tokenizer_path": "deepseek-ai/DeepSeek-V3",
+            "dtype": "bfloat16",
+            "opt_type": "adam_pax",
+            "attention": "flash",
+        },
+        xla_flags=(
```

Comment:
In my recent dropping experiments, I noticed those flags may be helpful (1-2% MFU): https://screenshot.googleplex.com/3zAXSRMNQN6ufoL

1) HOST_OFFLOAD_FLAGS
2) LAYOUT_FOR_ALL_REDUCE_SCATTER
3) ENABLE_SPARSECORE_OFFLOADING_FOR_RS_AG_AR (does not help)
4) also worth to try if `remat_policy=custom decoder_layer_input=offload out_proj=offload query_proj=offload key_proj=offload value_proj=offload` can fit

Reply:
Thanks @RissyRan sharing all this info. Are you seeing perf improvement on v6e? 
We will retune all those flags for v5p. this is the baseline setup.

Reply:
Yes, I did observe the perf improvement so sharing it here :) Could you just run a test with FSDP + EP, it should give a better perf. 

ici_expert_parallelism=8 or 64, and ici_fsdp_parallelism=64 and 8 respectively. Other configs could keep the same. With dropping, EP sharding will boost the MFU.